### DISCUSSION TITLE
Empty dataframe returned upon using select_features() method

### DISCUSSION THREAD

**Discussion by Bling16** [blue-yonder/tsfresh#1036] [Q&A]
I am currently using tsfresh for one of my projects. Upon using extract_features(), I get a dataframe of 46 rows × 5439 columns (46 different ids). However, when I run the select_features() method, I get a dataframe of shape (46,0). I am not able to figure out the error occurring in the code and would appreciate if someone would help me out with it.

The main dataframe has only numerical values in the columns. The target column y (for select_features() method) is a Pandas Series having binary values for the target.

**Comment by nils-braun** [comment]
It looks like the feature selection just did not find any relevant features. 
You can test the selection pipeline by e.g. doing the feature extraction only and then adding a very trivially correlated feature, like the target column itself, to the dataset. Then run only the feature selection. You newly added feature should definitely survive the selection.
